docs/source/index.rst

```diff
@@ -4,16 +4,16 @@
 NumPyro documentation
 =====================
 
-.. toctree::
-   :maxdepth: 1
-
-   getting_started
-
 .. toctree::
    :maxdepth: 1
    :caption: API and Developer Reference
 
-   api
+   getting_started
+   primitives
+   distributions
+   infer
+   handlers
+   contrib
```

Comment:
This is to make the sidebar closer to Pyro docs. In addition, by removing the `api` level, more headers can be accessed through the sidebar.